pages/DSNP/Messages/Overview.md

```diff
@@ -65,94 +64,112 @@ For details on how messages are serialized, see [DSNP Message Serialization](/DS
 #### Broadcast
 a public post (was Announcement)
 
-| field     | description | type |
-|-------    |-------------| ----|
-| inReplyTo | messageID replied to | bytes32
+| dsnpData field | description | type |
+| ------------- |------------- | ---- |
+| fromAddress | ID of the sender | bytes20
 | messageID | keccak-256 hash of content stored at uri |  bytes32
-| uri       | content uri | bytes
+| uri       | content uri | bytes[]
 
 
-**NOTE** If origin broadcasts and replies stay the same type, inReplyTo is allowed to be blank.  If replies are separated into their own type, the inReplyTo field here will be dropped.
+#### Reply
+a public reply post
+
+| dsnpData field | description | type |
+| ------------- |------------- | ---- |
+| inReplyTo | ID of the message the reply references |  bytes32
+| messageID | keccak-256 hash of content stored at uri |  bytes32
+| fromAddress | ID of the sender | bytes20
+| uri       | content uri | bytes[]
+ 
 
 #### Drop
 a dead drop message
 
-| field | description | type
-|-------|-------------| ---|
-| ddid | dead drop id |  bytes
-| uri  | content uri  |  bytes
+| dsnpData field | description | type |
+| ------------- |------------- | ---- |
```

Comment:
I learned recently you just need three dashes between pipes to cause it to be rendered as a table.

Reply:
@shannonwells I wonder if we want a lint formatter to do the table formatting for us... Make them look right every time. Or I am also ok with a standard of just three `---` and spaces before and after. So like:
```markdown
| title | spaces around |
| --- | --- |
| content | also has spaces around |
```